foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/CountAction.java

```diff
@@ -105,4 +104,15 @@ public ActionResponse execute(CountRequest parameter) throws QueryStoreException
                     "Error running count query.", e);
         }
     }
+
+    @Override
+    protected boolean validate() {
+        getParameter().setTable(ElasticsearchUtils.getValidTableName(getParameter().getTable()));
```

Comment:
Method name validate implies nothing should be changed inside the method. It should work as pure validator